whylog/constraints/verifier.py

```diff
@@ -172,7 +172,7 @@ def single_constraint_not(cls, clues_lists, effect, constraint, constraint_manag
         """
         clues_lists = cls._construct_proper_clues_lists(clues_lists)
         for combination in cls._clues_combinations(clues_lists):
-            if cls._verify_constraint(combination, effect, constraint, constraint_manager):
+            if cls._verify_constraint(combination, effect, 0, constraint, constraint_manager):
```

Comment:
Add some explanation that this 0 idx came form assumption that has just one constraint. Also some const for that.